### PR TITLE
fix(components): style error when component import order is unknown

### LINKED ISSUE #5667: [Bug Report] In manually import, ".el-icon" & ".el-input__icon" style error
### Element Plus version
1.3.0-beta.9

### OS/Browsers version
MacOS 11.6.2 / Chome 96.0.4664.55

### Vue version
3.2.29

### Reproduction Link
https://github.com/TothingWay/test-ep

### Steps to reproduce
1. use unplugin-element-plus import element-plus components
2. Add "show-password"  to el-Input component
3. You can see the eye icon style error
4. I found that is .el-icon style overrides the .el-input__icon style

### What is Expected?
The eye icon style is displayed correctly

### What is actually happening?
The eye icon style display error

### LINKED ISSUE #5968: [Bug Report] ElNotification样式错乱
### Element Plus version
2.0.1

### OS/Browsers version
latest

### Vue version
3.2.30

### Reproduction Link
https://github.com/tfn290181662/element-plus-test

### Steps to reproduce
运行项目，手动路由到http://localhost:3000/#/about，即可复现

### What is Expected?
ElNotification正常显示

### What is actually happening?
ElNotification样式错乱

### PR DESCRIPTION
**Pull request by emojiiii**
Please make sure these boxes are checked before submitting your PR, thank you!

- [ ] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [ ] Make sure you are merging your commits to `dev` branch.
- [ ] Add some descriptions and refer to relative issues for your PR.

closed #5667

### CODE REVIEW

File: packages/components/alert/src/alert.vue [hunk 1]
```diff
@@ -26,12 +26,12 @@
         <template v-if="closable">
           <div
             v-if="closeText"
-            :class="[ns.e('closebtn'), ns.is('customed')]"
+            :class="[ns.e('closeBtn'), ns.is('customed')]"
```
Comment (sxzz) [inline packages/components/alert/src/alert.vue:29]: How about change class name to `close-btn`
```suggestion
            :class="[ns.e('close-btn'), ns.is('customed')]"
```

Comment (emojiiii) [reply packages/components/alert/src/alert.vue:29]: because `closeBtn` is used elsewhere..

**Review by jw-foss — COMMENTED** [review]
Can we change it to `close-btn` instead of `closeBtn` for those are the same we should update them in the future.

File: packages/components/alert/src/alert.vue (hunk 1 above)
Comment (emojiiii) [reply packages/components/alert/src/alert.vue:29]: ok!

**Review by jw-foss — APPROVED** [review]
LGTM

**Pull request merged**